dashboard/config/locales/resources.th-TH.json

```diff
@@ -7971,7 +7974,7 @@
         },
         "classifying_fish/how-ai-works/2023": {
           "name": "การจำแนกปลา",
-          "url": "https://docs.google.com/document/d/1gQWtamQ3YBk-gJnmw_z3QWNLfzdrPyaUjS4ZbtfAvEs/view",
+          "url": "https://docs.google.com/document/d/1-jk6nypSNHxokQP3DJRgITQdvjkNGFzNsP2wYM9vfeg/",
```

Comment:
@castro-jorge flagging these links to resources. Please double-check these updates are expected. I have no access to the documents, and I would assume students/teachers won't either.

Reply:
@mgc1194 our Turkish partner updated the share settings of all the documents.